vnext/Microsoft.ReactNative/Fabric/Composition/Modal/WindowsModalHostViewComponentView.cpp

```diff
@@ -100,6 +244,34 @@ void WindowsModalHostComponentView::updateLayoutMetrics(
 
         float offsetX = static_cast<float>(offset.x / m_layoutMetrics.pointScaleFactor);
         float offsetY = static_cast<float>(offset.y / m_layoutMetrics.pointScaleFactor);
+
+        winrt::com_ptr<IDWriteTextFormat> spTextFormat;
```

Comment:
It take it all of this text rendering here is just to show that you can have some content inside? If so, maybe more clearly identify it as temporary with a comment.

Reply:
From the video of the demo, didn't look like it was rendering inside the window. But that's the next goal?

Reply:
That's the next goal but I'll add a comment to say this is temporary just to make sure this file can draw on the main window